app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+
+    if command_parts[0] != "mv" or len(command_parts) != 3:
+        raise ValueError("Incorrect command input.")
+
+    old_file, new_file = command_parts[1:]
+
+    if new_file.endswith("/"):
```

Comment:
The code tries to avoid treating new_file as a directory if it ends with a /, but directories don’t always have a trailing slash. Instead of just checking for /, use os.path.isdir(new_file) to properly check if it's a directory.